tapdb/assets_store_test.go

```diff
@@ -2204,6 +2205,44 @@ func TestTransferOutputProofDeliveryStatus(t *testing.T) {
 	require.Equal(
 		t, randBlockHash[:], assetTransfers[0].AnchorTxBlockHash,
 	)
+
+	// Let's insert a burn.
```

Comment:
How about we make another isolated unit test for this? This exiting test is already rather long/large.